app/views/Import.js

```diff
@@ -44,6 +44,7 @@ class ImportScreen extends Component {
   }
 
   render() {
+    var counter = 0;
```

Comment:
I believe if you're running the pre-commit hook eslint is going to want you to change this to 'let'.